lib/activity_runtime_api.rb

```diff
@@ -141,23 +142,26 @@ def self.publish_sequence(hash, user)
       investigation = Investigation.create(
         :name => hash["name"], :description => hash['description'],
         :abstract => hash['abstract'], :user => user)
+      all_student_reports_enabled = true
       hash['activities'].each_with_index do |act, index|
         activity_from_hash(act, investigation, user, index)
+        all_student_reports_enabled &&= (act.has_key?(:student_report_enabled) ? act[:student_report_enabled] : true)
```

Comment:
Because this won't be obvious what the logic is when we look at this in a few months. It should at least have a test that demonstrates how it only take one disabled student report, to disable it for the whole sequence.

Reply:
I have a test for this already - https://github.com/concord-consortium/rigse/blob/13837c5a9e48682fd1a48c99871c9598219ec381/spec/libs/activity_runtime_api_spec.rb#L504-L508

Reply:
Which I just see is failing... :)